i18n/fr.yml

```diff
@@ -0,0 +1,14 @@
+invoice:
+  invoice: Facture
+  prepared_for: Émise à
```

Comment:
Oh yes, I see why it would come out this way in Translate. With a literal translation, "Émise à" will be read as "Issued to" (it would indeed mean "Issued on" if a date followed). "Par" means "By" in the context of something produced.

I'd stand with my current suggestions, or make it more balanced by using _Émise à_ and _Préparée par_ ("Issued to", and "Prepared by"). "Par" makes for a very short label if it's right-aligned.

[Here's a verification translation that includes nouns.](https://translate.google.com/#fr/en/%C3%89mise%20%C3%A0%20Louis-Andr%C3%A9%2C%0A%0APr%C3%A9par%C3%A9e%20par%20Jonnie)